### DISCUSSION TITLE
Graphic display of G-force data?

### DISCUSSION THREAD

**Discussion by Corradopsi** [BimmerGestalt/AAIdrive#733] [Ideas]
I like the new app with data pulled in from car sensors. I see that there is a G-force sensor reading data. Is there a way to get a graphic display of that like the M cars have, or even better have it added to the factory sport gauges screen? Or is there a coding option to add this that I am unaware of?

**Comment by hufman** [comment]
I don't know any options for changing the factory sport gauges screen.
Showing a graphical display is harder, AAIdrive would have to render an image and send it over the Bluetooth app connection to the car. While it would update faster than the map screen, I feel it would not be satisfying anyways.
The legacy Mini Connected Sports Display on iPhone relied on uploading an image pack and changed the image widget to display different images from the pack, but these image packs are locked by BMW and can't be modified or even used by any other apps.

**Comment by labanauskas** [comment]
> I like the new app with data pulled in from car sensors. I see that there is a G-force sensor reading data. Is there a way to get a graphic display of that like the M cars have, or even better have it added to the factory sport gauges screen? Or is there a coding option to add this that I am unaware of?

Maybe FDL coding of HU will solve. You need to use bimmercode or bimmerutility to check if that FDL is possible. I'll look into my f30 lci with id6
